<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="createRequestRadio">
          <property name="text">
           <string>Create Request:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="createRequestGBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="slaveIDlbl">
               <property name="text">
                <string>Slave ID (0-247):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="slaveID">
               <property name="maximum">
                <number>247</number>
               </property>
               <property name="displayIntegerBase">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="funcCodeLbl">
               <property name="text">
                <string>Function Code:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="functionCode"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="regAddrLbl">
               <property name="text">
                <string>Reg. address:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="regAddress">
               <property name="maximum">
                <number>65535</number>
               </property>
               <property name="displayIntegerBase">
                <number>16</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="regQuantLbl">
               <property name="text">
                <string>Reg. Quantity:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="regQuantity">
               <property name="maximum">
                <number>65535</number>
               </property>
               <property name="displayIntegerBase">
                <number>16</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="CRClbl">
               <property name="text">
                <string>CRC16:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CRC">
               <property name="text">
                <string>----</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="typeRequestRadio">
          <property name="text">
           <string>Type Request:</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="typeRequestLineEdit">
             <property name="text">
              <string/>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="sendEveryChBox">
          <property name="text">
           <string>Send every:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sendEvery">
          <property name="suffix">
           <string> ms</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionClear"/>
   <addaction name="actionSettings"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/action/img/connect.png</normaloff>:/action/img/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/action/img/disconnect.png</normaloff>:/action/img/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/action/img/wroom.png</normaloff>:/action/img/wroom.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/action/img/settings.png</normaloff>:/action/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/action/img/close.png</normaloff>:/action/img/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
